media/js/base/mozilla-client.js

```diff
@@ -239,7 +239,7 @@ if (typeof Mozilla === 'undefined') {
         };
 
         var onRetrieved = function (accurate, version, channel, distribution) {
-            document.removeEventListener('mozUITourResponse', listener);
+            document.removeEventListener('mozUITourResponse', listener, false);
```

Comment:
This was throwing an error in really old Firefox versions as `useCapture` wasn't always optional. I'm setting it explicityle here to the default value, which is `false`.